tools/cosmovisor/cmd/cosmovisor/main.go

```diff
@@ -3,16 +3,22 @@ package main
 import (
 	"context"
 	"os"
+	"time"
 
 	"cosmossdk.io/log"
+
 	cverrors "cosmossdk.io/tools/cosmovisor/errors"
 )
 
 func main() {
-	logger := log.NewLogger(os.Stdout).With(log.ModuleKey, "cosmovisor")
-	ctx := context.WithValue(context.Background(), log.ContextKey, logger)
+	// error logger used only during configuration phase
+	logger := log.NewLogger(
+		os.Stderr,
+		log.ColorOption(false),
+		log.TimeFormatOption(time.Kitchen),
+	).With(log.ModuleKey, "cosmovisor")
 
-	if err := NewRootCmd().ExecuteContext(ctx); err != nil {
+	if err := NewRootCmd().ExecuteContext(context.Background()); err != nil {
```

Comment:
Have you tested the init command? It tries to get the log from the context, so I do not think this will work.

Reply:
```shell
❯ ./cosmovisor init
Error: accepts 1 arg(s), received 0
Usage:
  cosmovisor init <path to executable> [flags]

Flags:
  -h, --help   help for init

6:27PM ERR error="accepts 1 arg(s), received 0" module=cosmovisor

```

Reply:
That does not show it works, cobra hasn't executed `RunE` because an argument was missing.

Reply:
i see what you mean, i somehow missed init command.